test/xdr/transactions/fee_bump_innert_tx_test.exs

```diff
@@ -13,7 +13,8 @@ defmodule StellarBase.XDR.Operations.FeeBumpInnerTxTest do
     Operation,
```

Comment:
@CristhianRodriguezMolina @FelipeGuzmanSierra the file name has an extra letter "t" `fee_bump_inner**t**_tx_test.exs`